user_docs/en/changes.md

```diff
@@ -2,21 +2,42 @@
 
 ## 2024.4
 
-### Important notes
+This release includes a number of improvements in Microsoft Office, braille, and document formatting.
+There are also a number of other new features and fixes across the board.
+
+In Word or Excel, it is now possible to double-press the comment gesture to read the comment or note in a browsable dialog.
+You can now use the review cursor selection command to select text in PowerPoint.
+NVDA also no longer brailles garbage characters when showing row or column header text in tables in Word when using the object model.
+
+NVDA can now be configured to report font attributes in speech and braille separately.
+
+A new setting has been added to configure the timeout to perform a multiple press gesture, such as the report time/date command.
+
+You can now configure how NVDA shows text formatting in braille, and set NVDA to show the start of paragraphs in braille.
+NVDA can now speak the character at the cursor when performing a braille cursor routing action.
+Cursor routing reliability has been improved, and support for routing keys in PowerPoint has been added.
+All lines of cells will now be used when using a multi-line braille display via HID braille.
+The Seika Notetaker driver now correctly generates braille input for space, backspace, and dots with space/backspace gestures.
+NVDA is no longer unstable after restarting NVDA during an automatic Braille Bluetooth scan.
+
+eSpeak NG has been updated, adding support for the Faroese and Xextan languages.
+
+There have also been a number of fixes, including to the on-demand speech mode and mouse tracking in Firefox.
```

Comment:
Changed the order, just so it doesn't sound like it could mean "fixes to the mouse tracking in Firefox". Yeah, now *that* is nitpicking.

```suggestion
There have also been a number of fixes, including to mouse tracking in Firefox, and the on-demand speech mode.
```

Reply:
I love nitpicks that actually add something, which this one certainly did :)